benchmark.py

```diff
@@ -533,20 +534,23 @@ def run_throughput_test(query_root, data_dir, result, host, port, db_name, user,
         result = Result("ThroughputRStream")
         processes = []
         for i in range(num_streams):
+            # queries
+            stream_num = i + 1
+            print("Running stream #%s" % str(stream_num))
             p = Process(target=run_throughput_inner,
                         args=(query_root, data_dir, host, port, db_name, user, password,
-                                i+1, q))
+                              stream_num, q))
             processes.append(p)
             p.start()
-        for i in range(num_streams):
+            # refresh functions
```

Comment:
Maybe I am wrong but this does not seem correct. `p.start()` is async. However, calling refresh functions is synchronous. You would not start stream 2 unless refresh functions finish. I don't think you can combine these loops.

Reply:
OK, now I got the idea :)